test/user.test.js

```diff
@@ -2124,6 +2124,40 @@ describe('User', function() {
         },
       ], done);
     });
+
+    it('keeps sessions AS IS if partial update is done with age', function(done) {
```

Comment:
Please move the test closer to existing `keeps sessions AS IS` tests and rename it to mention `updateAll`, e.g. `keeps sessions AS IS if non-email property is changed using updateAll`.